### ISSUE TITLE
Issue with the README and copyrights

### ISSUE THREAD

**Issue by Zorrototo** [OpenGamePanel/Theme-MasterControlProgram#1]
> Users may edit the item, although users are reminded the Master Control Program - MCP logo is copyrighted, nor can users remove the OGP Copyright. Users must not re-distribute the theme's copy for free or paid.

That is not how GPL licenses work. [OGP license](https://github.com/OpenGamePanel/OGP-Website/blob/master/COPYING)

If you don't want to make your work under the OGP license (which you HAVE TO) then you should not expand OGP.

Reminder:

### OpenGamePanel/OGP-Website is licensed under the GNU General Public License v2.0

The GNU GPL is the most widely used free software license and has a strong copyleft requirement. When distributing derived works, the source code of the work must be made available under the same license. There are multiple variants of the GNU GPL, each with different requirements.

**Permissions**

_Commercial use
Modification
Distribution
Private use_ 

**Limitations**

_no Liability
no Warranty_

**Conditions**

_License and copyright notice
State changes
Disclose source
Same license_ 

Also on a side note, you could at least credit the theme author from the theme you used as a base for your theme modification.

**Comment by own3mall** [comment]
Thanks for pointing this out.  I just focused on getting it added for the creators.  Removed invalid info.

**Comment by Zorrototo** [comment]
:+1: it is just to avoid misunderstanding in the future, I myself may not understand all about the license but because of past experiences I prefer staying vigilant.

**Comment by P34NUT85** [comment]
Hi guys apologies for the delayed response to this, and also apologies for any inconvenience this has caused. 

It is not our intention to duplicate the work exactly and claim it as our own, but simply show the process we have gone through from start to finish. As explained on the forum thread on the OGP Forum, I personally understood from the licence that we could not copy the work like for like and claim it as our own, or charge others for its use. For example Renaming OGP to something like "The Hub" and the charging of a fee similar to what WHMC does as a price tag. 

I can understand people being a bit jumpy with it, as we are admittedly slow in progress with our development with this, due to it being a personal project with little funding. I dont intend to challenge the decision you guys have made, as mentioned in the email sent to me "lets try to avoid conflict" as even myself coming into this I have noticed there is or was "little" activity with regard's to OGP vs LinuxGSM and we just want to contribute while we look to develop something for ourselves and help OGP benefit from it. 

So from me personally ill just say 


Apologies for the inconvenience we caused and we will adapt the way we do things in the future ^_^
